packages/core/cms/sections.json

```diff
@@ -1,4 +1,283 @@
+            }
+          }
+        },
+        "search-products": {
+          "title": "Suggested Products",
+          "type": "object",
+          "properties": {
+            "title": {
+              "title": "Title",
+              "type": "string",
+              "default": "Suggested Products"
+            },
+            "max-items": {
+              "title": "Maximum Number of Suggested Products",
+              "type": "integer",
+              "default": 5
+            }
+          }
+        }
+      }
+    }
+  },
+  {
+    "name": "Navbar",
+    "schema": {
+      "title": "Navbar",
+      "type": "object",
+      "description": "Navbar configuration",
+      "required": ["image"],
+      "properties": {
+        "logo": {
+          "title": "Logo",
+          "type": "object",
+          "properties": {
+            "src": {
+              "title": "Image",
+              "type": "string",
+              "widget": {
+                "ui:widget": "media-gallery"
+              }
+            },
+            "alt": {
+              "title": "Alternative Label",
+              "type": "string"
+            }
+          }
+        },
+        "sign-in-button": {
+          "title": "Sign In Button",
+          "type": "object",
+          "properties": {
+            "icon": {
+              "title": "Icon",
+              "type": "object",
+              "properties": {
+                "icon": {
+                  "title": "Icon",
+                  "type": "string",
+                  "enumNames": ["User"],
+                  "enum": ["User"],
+                  "default": "User"
+                },
+                "alt": {
+                  "title": "Alternative Label",
+                  "type": "string",
+                  "default": "User"
+                }
+              }
+            },
+            "label": {
+              "title": "Call to Action",
+              "type": "string",
+              "default": "Sign In"
+            }
+          }
+        },
+        "cart-icon": {
+          "title": "Cart Icon",
+          "type": "object",
+          "properties": {
+            "icon": {
+              "title": "Icon",
+              "type": "string",
+              "enumNames": ["Shopping Cart"],
+              "enum": ["ShoppingCart"],
+              "default": "ShoppingCart"
+            },
+            "alt": {
+              "title": "Alternative Label",
+              "type": "string",
+              "default": "Shopping Cart"
+            }
+          }
+        },
+        "navigation": {
+          "title": "Navigation",
+          "type": "object",
+          "properties": {
+            "regionalization": {
+              "type": "object",
+              "title": "Regionalization",
+              "properties": {
+                "enabled": {
+                  "type": "boolean",
+                  "title": "Use Regionalization?",
+                  "default": true
+                },
```

Comment:
We don't have this option in frontend, should we create a task to handle it?

Reply:
Oh, it is true, I created in this link as subtask of navigation https://vtex-dev.atlassian.net/browse/FS-983